StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/BottomSheetViewController.swift

```diff
@@ -66,6 +80,73 @@ class BottomSheetViewController: UIViewController, BottomSheetPresentable {
         contentViewController = toVC
         return popped
     }
+    func addBlurEffect(animated: Bool, completion: @escaping () -> Void) {
+        if let containingSuperview = self.view.superview {
```

Comment:
Not sure if this would always cover the entire screen, depending on where we're presenting the BottomSheetViewController from.

Reply:
I believe the BottomSheetViewController's view's parent is a uitransition view, which is expected to cover the entire screen, so I think we are good here, but if Apple changes something, then right -- may not always work.